source/pkcs11.c

```diff
@@ -12,3 +20,359 @@
 #define CK_DECLARE_FUNCTION_POINTER(returnType, name) returnType(CK_PTR name)
 #define CK_CALLBACK_FUNCTION(returnType, name) returnType(CK_PTR name)
 #include <aws/io/private/pkcs11/v2.40/pkcs11.h>
+
+/* Return c-string for PKCS#11 CKR_* contants */
+const char *s_ckr_str(CK_RV rv) {
+    /* clang-format off */
+    switch (rv) {
```

Comment:
How do you get this list?

Reply:
I copy-pasted from [pkcs11t.h](https://github.com/oasis-tcs/pkcs11/blob/0c3ecef6c90dcd8249ba64cc99d50d9f8f73124e/published/2-40-errata-1/pkcs11t.h#L1039)

Reply:
so with a new version, we would need to update this as well.